_pages/fsa/dynamicforms/core/basic-form-validations.md

```diff
@@ -0,0 +1,285 @@
+---
+title: Basic Form Validations
+---
+
+## Overview
+Dynamic Forms library provides some basic validator functions by default. This page will explain how to use, override or add validations in Dynamic Forms library for end customers. Following things should be covered:
+
+- [How to invoke validation from JSON ?](#how-to-invoke-validation-from-json)
+- [Which validations exists out of the box ?](#existing-validations) 
+- [How all this works ?](#how-all-this-works)
+- [How to override existing validation in custom App ?](#how-to-override-existing-validation-in-custom-app)
+- [How to add new validation in custom App and invoke that from JSON ?](#how-to-add-new-validation-in-custom-app-and-invoke-that-from-json) 
+
+## How to invoke validation from JSON
+
+In order to specify validations for certain field, JSON schema is enhanced by exposing property called "validations".  Property represents array of complex objects which are translated in dynamic forms validation functions and they consist of two properties:
```

Comment:
"which are translated **in** dynamic" => "which are translated **into** dynamic"